### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by NoahGallant-MUN**
Added translation tags where missing. #570

### CODE REVIEW

File: oceannavigator/frontend/src/components/CoordInputPanel.jsx [hunk 1]
```diff
@@ -62,7 +62,7 @@ export default class CoordInputPanel extends React.Component {
               onChange={e => this.onChange(e.target, DataInput.LATITUDE)}
             />
             <br />
-            <ControlLabel>{_("Longtitude")}: </ControlLabel>
+            <ControlLabel>{_("Longittude")}: </ControlLabel>
```
Comment (htmlboss) [inline oceannavigator/frontend/src/components/CoordInputPanel.jsx]: Typo?

File: oceannavigator/frontend/src/components/WarningBar.jsx [hunk 2]
```diff
@@ -24,7 +24,7 @@ export default class WarningBar extends React.PureComponent {
     if(this.props.showWarningInfo){
       return (
         <Alert bsStyle="warning" onDismiss={this.handleDismiss}>
-              The Ocean Navigator will go down for brief maintenance at 12:45 NST
+              {_("The Ocean Navigator will go down for brief maintenance at 12:45 NST")}
```
Comment (htmlboss) [inline oceannavigator/frontend/src/components/WarningBar.jsx]: Could we throw in a date too?

File: oceannavigator/frontend/src/components/CoordInputPanel.jsx (hunk 1 above)
Comment (NoahGallant-MUN) [reply oceannavigator/frontend/src/components/CoordInputPanel.jsx]: wow, when you fix a typo with another typo

File: oceannavigator/frontend/src/components/WarningBar.jsx (hunk 2 above)
Comment (NoahGallant-MUN) [reply oceannavigator/frontend/src/components/WarningBar.jsx]: Sure, but this section is supposed to be updated when your doing a rollout.
Seperate from this issue?

File: oceannavigator/frontend/src/components/CoordInputPanel.jsx (hunk 1 above)
Comment (NoahGallant-MUN) [reply oceannavigator/frontend/src/components/CoordInputPanel.jsx]: fixed

File: oceannavigator/frontend/src/components/WarningBar.jsx (hunk 2 above)
Comment (NoahGallant-MUN) [reply oceannavigator/frontend/src/components/WarningBar.jsx]: Added the date, but we will have to update the current public branch for this to actually be usefule

**Pull request merged**